be/src/storage/lake/tablet.h

```diff
@@ -139,10 +139,17 @@ class Tablet {
 
     int64_t version_hint() const { return _version_hint; }
 
+    void set_version(int64_t version) { _version = version; }
+
+    int64_t get_version() { return _version; }
+
 private:
     TabletManager* _mgr;
     int64_t _id;
     int64_t _version_hint = 0;
+
+    // used for auto increment only
+    int64_t _version = 1;
```

Comment:
Not need this if we can get latest tablet meta from cache?

Reply:
done